graphics/fonts/macfont.cpp

```diff
@@ -485,7 +485,6 @@ int MacFONTFont::getKerningOffset(uint32 left, uint32 right) const {
 bool dododo;
```

Comment:
Why is it still here? Look, you declare it, you assign it a value, but then you do not use it anywhere!

Reply:
so this was already there in Macfont.cpp original file. It was used in maginfy gray for debugging but since that function is no longer there, I will remove it